playbooks/roles/airship-deploy-tempest/files/tempest_setup_network.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -xe
+
+export OS_CLOUD=openstack
+
+export OSH_EXT_NET_NAME="public"
```

Comment:
does the ext network have to actually exist on the host for the test to succeed? If true, do we need to make this user input in extravars?